pkgs/top-level/all-packages.nix

```diff
@@ -35902,15 +35902,18 @@ with pkgs;
   kodiPackages = recurseIntoAttrs (kodi.packages);
 
   kodi = callPackage ../applications/video/kodi {
+    ffmpeg = ffmpeg_6-full;
```

Comment:
Why do we have to fix the version here? Is this because we need the full package?

Reply:
we need version 6 specifically

i assume we want the `full` variant

Reply:
Kodi also builds with "small", and even "headless" successfully, and compile-time configuration stays the same.
There are no build-time dependencies on "full".

Ffmpeg's derivation [discourages](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/ffmpeg/generic.nix#L30) the use of "full" variant.
Greping the features "full" (`withFullDeps`) variant enables, it seems highly unlikely an average Kodi setup is going to ever need them as a runtime dependency: those are mostly alternatives to already-enabled stuff or something obscure.

Maybe we should keep the regular ffmpeg variant here, "full" doesn't seem to provide any additional value.